config/routes.rb

```diff
@@ -55,7 +55,9 @@
   end
 
   namespace :admin do
-    get '/dashboard', to: "dashboard#show"
+    get '/', to: "dashboard#show"
```

Comment:
I was gonna ask if we we're gonna make those minor changes, but looks like you already did it 👍

Reply:
Gotta do the same for merchant as well